lib/assessment.js

```diff
@@ -4,24 +4,31 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(arg) {
+  min = math.min();
 };
+//I used mdn
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  return math.sum();
 };
 
 // Question 3
 // Write a function that returns an array of the specified length
 // with all the elements set to defaultValue
 var newArray = function newArray(length, defaultValue) {
+  var newArray = [defaultValue];
```

Comment:
Finally, on naming. Naming this variable `newArray` works. The challenge is that it denies you access to another variable called `newArray` that points to the function that is currently executing. It won't throw an error when you do it here, but you should be conscious that you are making your function incapable of referencing itself when you create a variable of the same name. However, the greatest damage is done to readability. Someone else reading your code (including yourself in the future) will find this confusing.